### ISSUE TITLE
createBottomTabNavigator broken on iPad iOS 11.4

### ISSUE THREAD

**Issue by JoseVf** [react-navigation/react-navigation#4579]
### Current Behavior

- Tab icons on iPad are not showing properly as below image shows
![screen shot 2018-06-26 at 8 50 38 am](https://user-images.githubusercontent.com/22102736/41916812-7547c2fe-791e-11e8-9de9-71ac1458dfb1.png)
but works on iPhone
![screen shot 2018-06-26 at 9 28 20 am](https://user-images.githubusercontent.com/22102736/41919092-4c100c66-7923-11e8-8d3b-6716608471d3.png)

### Expected Behavior

- Tab icons display correctly on iPad and iPhone

### How to reproduce

- Actually can't reproduce issue on Expo 😕

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 2.4.1
| react-native     | 0.55.4
| node             | 9.3.0
| npm     | 5.6.0

**Comment by proutek** [comment]
Same problem here. ipad mini (IOS 11.3).
@brentvatne It's not reproducible as snack because there are only IOS phone devices.
![bottomtabnav-ipadmini](https://user-images.githubusercontent.com/21219132/47773558-c1ff6b80-dcea-11e8-8d95-156a29078795.png)

**Comment by brentvatne** [comment]
create a new issue and share a screenshot along with a snack - you can run it on your device by installing the expo client and scanning the qr code in the “run” modal in snack

**Comment by atulmy** [comment]
Use following configuration:

```
createBottomTabNavigator(routes, {
  tabBarOptions: {
    style: {
      backgroundColor: grey,
      paddingVertical: 10
    },
    tabStyle: {
      flexDirection: 'column',
      alignItems: 'center',
      justifyContent: 'center'
    },
    labelStyle: {
      marginLeft: 0
    }
  }
})
```

**Result**
From

<img width="567" alt="screenshot 2018-11-09 at 6 27 03 pm" src="https://user-images.githubusercontent.com/685786/48263745-29d15700-e44d-11e8-811b-90f6dbbb8e68.png">

To

<img width="560" alt="screenshot 2018-11-09 at 6 27 28 pm" src="https://user-images.githubusercontent.com/685786/48263755-2d64de00-e44d-11e8-973e-ffe06b31bfda.png">

**Comment by brentvatne** [comment]
if you want to open a pr to disable horizontal mode that's fine -- https://github.com/react-navigation/react-navigation-tabs

otherwise, see this thread for more discussion https://github.com/react-navigation/react-navigation-tabs/issues/68

tl;dr: just use tabBarLabel: 'some-string', and not tabBarLabel: <Text>someString</Text> and the layout works as expected

**Comment by MarangoniEduardo** [comment]
tabBarOptions -> labelPosition - Where to show the tab label in relation to the tab icon. Available values are beside-icon and below-icon. Defaults to below-icon on mobile and beside-icon on tablets.
